cmd/podman/cleanup.go

```diff
@@ -37,6 +37,7 @@ func init() {
 
 	flags.BoolVarP(&cleanupCommand.All, "all", "a", false, "Cleans up all containers")
 	flags.BoolVarP(&cleanupCommand.Latest, "latest", "l", false, "Act on the latest container podman is aware of")
+	flags.BoolVar(&cleanupCommand.Rm, "rm", false, "After cleanup, remove the container entirely")
```

Comment:
OK, completely a personal twitch, but could you change the name from 'Rm' to 'Remove'.  Variable names with less than 3 chars make me twitch, especially when their scope is more than a couple of lines.

Reply:
Agree make this Remove.